README.md

```diff
@@ -1,2 +1,54 @@
+
+SATRE will compare openly available UK TREs hosting health, manufacturing, commercial, science and humanities data and bring them into alignment with a standardised TRE reference architecture (or structural template). 
+The development teams at the University of Dundee’s Health Informatics Centre and the Alan Turing Institute, fully supported by their infrastructure partners, will lead the reference alignment in collaboration with a consortium of higher education, charity and industry organisations.
+
+The reference architecture and its implementation will be informed and strengthened by a programme of community building and engagement with DARE UK and partner working groups, other stakeholders and inclusive public representation.
+SATRE outputs will include an informed TRE reference technical specification and a collection of educational media and detailed reports – all supporting DARE UK’s aim of a coordinated national data research infrastructure.
+
+## Community-Driven Approach
+
+[![Slack](https://img.shields.io/badge/Slack-Join%20Our%20Channel-blue?logo=slack)](https://satreproject.slack.com)
+
+The SATRE project values the importance of equity, diversity, and inclusion (EDI) in fostering a kind and supportive environment for all participants.
+We are committed to making the project as inclusive as possible and appreciate the unique perspectives and input that each community member brings.
+
+While there are core members funded by DARE to contribute to this project, our primary goal is to involve as many members of the community as possible.
+We strongly believe that the best results come from collaboration, diverse perspectives, and fostering a culture of kindness and mutual respect.
+
+## Getting Started
+To participate in the SATRE project, please follow these steps:
+
+1. **Join the community**: Connect with other participants through our mailing list, forum (GitHub Discussions), or [Slack channel](slack.url). Introduce yourself and share your experience and interests related to TREs. The Community meets regularly:
+
+| Meeting | Description | Time | Link
+|----------|----------|----------|----------|
+| Bi-weekly Collaboration Cafe  | Meeting and decide on topics and work on them via co-working breakout rooms  | Every 1st and 3rd X at Y  |[zoom link please fill]()|
+| Monthly Open Roadmap Review  | Core funded team to provide updates to the Open Roadmap based on community contributions  | Every 3rd week of the month at Y  |[zoom link please fill]()|
+
+2. **Contribute**: Review open issues, join SATRE meetings, and provide your input on the design, implementation, and best practices for TREs. To contribute to the project, create a pull request with your proposed changes or additions to the documentation. If you have a specific idea or proposal, feel free to open a new issue or start a discussion thread.
```

Comment:
I think it's too early to talk about pull requests, maybe just say open an issue or contribute to an existing one?